provisioning_templates/snippet/create_users.erb

```diff
@@ -0,0 +1,25 @@
+<%#
+kind: snippet
+name: create_users
+-%>
+# Create all users
+<% [@host.owner].flatten.each do |user| -%>
+<% username = user.login -%>
+useradd <%= username %>
+<% if user.ssh_authorized_keys && user.ssh_authorized_keys.any? -%>
+mkdir <%= username %>/.ssh
```

Comment:
SSH can be picky about `.ssh` and the key file permissions, should we enforce them here? Also would it be safer to run `mkdir -p`?